nixos/modules/services/misc/gitlab.nix

```diff
@@ -1276,6 +1276,7 @@ in {
     systemd.services.gitlab-config = {
       wantedBy = [ "gitlab.target" ];
       partOf = [ "gitlab.target" ];
+      bindsTo = [ "gitlab-db-config.service" ];
```

Comment:
This needs to be removed. It makes the test fail even with the fix, since it starts the database initialization when the config runs, which means the backup restoration fails.